dev/devicelab/manifest.yaml

```diff
@@ -357,6 +357,13 @@ tasks:
     required_agent_capabilities: ["linux/android"]
     flaky: true
 
+  build_benchmark:
+    description: >
+      Measures APK build performance.
```

Comment:
Can this be slightly more specific? Isn't build performance when config changes or something like that?

Reply:
Done